src/main.ts

```diff
@@ -92,6 +94,11 @@ code | condition
     operationIdFactory: (controllerKey: string, methodKey: string) => methodKey,
   });
 
+  const yamlSwaggerDoc = yaml.stringify(document);
+
+  // write the yaml swagger doc to the root project directory anytime the server starts
+  await fs.promises.writeFile("./swagger.yaml", yamlSwaggerDoc, "utf8");
```

Comment:
nit: we're already importing writeFile

```suggestion
  await writeFile("./swagger.yaml", yamlSwaggerDoc, "utf8");
```